Test/ArpTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using NUnit.Framework;
+using SharpPcap;
+using SharpPcap.LibPcap;
+
+namespace Test
+{
+    [TestFixture]
+    public class ArpTest
+    {
+        [Test]
+        public void TestArp([PcapDevices] DeviceFixture fixture)
+        {
+            var d = (LibPcapLiveDevice)fixture.GetDevice();
+            var arp = new ARP(d);
+
+            // timeout should not be null
+            Assert.IsNotNull(arp.Timeout);
+
+            // and we can set a timeout
+            arp.Timeout = new TimeSpan(0, 0, 2);
+
+            System.Net.NetworkInformation.PhysicalAddress localMAC = null;
+            foreach (var address in d.Addresses)
```

Comment:
Why not just use `d.MacAddress` ?

Reply:
Makes sense. I can't recall when that was added.